.doc_gen/metadata/sns_metadata.yaml

```diff
@@ -5,6 +5,13 @@ sns_GetTopicAttributes:
   synopsis: get the properties of an &SNS; topic.
   category:
   languages:
+    .NET:
+      versions:
+        - sdk_version: 3
+          github: dotnetv3/sns
```

Comment:
GitHub links are case sensitive, so this has to be `dotnetv3/SNS` or you get a 404.

Reply:
fixed